spec/swagger.js

```diff
@@ -0,0 +1,23 @@
+const swaggerJsdoc = require('swagger-jsdoc');
+
+// Import custom component schema definitions
+// This is used to have a single place to maintain schemas which will be reused
+const schemas = require('./schemas')
+
+const options = {
+  definition: {
+    openapi: '3.0.0',
+    info: {
+      title: 'E-Commerce Open Api v3.0.0 Specification',
+      version: '1.0.0',
+    },
+    components: {
+      schemas: schemas
```

Comment:
i ovdje nasem swagger-jsdoc paketu kazemo da ucita scheme iz schemas.js fajla... 

konkretno smo ih tamo eksportali kao JS objekt, i ovdje ih tako i koristimo, pošto je i definicija postavki swagger-jsdoc paketa također JS objekt